--- breakpad-0.1.4/src/client/linux/handler/exception_handler.cc.orig	2021-12-07 12:24:39.063827800 +0100
+++ breakpad-0.1.4/src/client/linux/handler/exception_handler.cc	2022-01-28 21:00:39.303807029 +0100
@@ -138,7 +138,7 @@ void InstallAlternateStackLocked() {
   // SIGSTKSZ may be too small to prevent the signal handlers from overrunning
   // the alternative stack. Ensure that the size of the alternative stack is
   // large enough.
-  static const unsigned kSigStackSize = std::max(16384, SIGSTKSZ);
+  static const unsigned kSigStackSize = std::max<unsigned>(16384, SIGSTKSZ);
 
   // Only set an alternative stack if there isn't already one, or if the current
   // one is too small.
